Makefile

```diff
@@ -174,6 +176,11 @@ tools: ## Install development tools.
 	# known issue https://github.com/kubernetes-sigs/kustomize/issues/3618
 	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v3@v$(KUSTOMIZE_VERSION))
 
+	GOBIN=$(BINDIR) go install github.com/norwoodj/helm-docs/cmd/helm-docs@v$(HELM_DOCS_VERSION)
+	curl -L -sS -o helm.tar.gz https://get.helm.sh/helm-v3.5.0-linux-amd64.tar.gz
```

Comment:
I doubt you missed using `HELM_VERSION`.
It may be preferable to use a pipe to avoid making temporary file like as: `curl http://... | tar xzvf - -C $(BINDIR)`

Reply:
Thanks! I fixed: https://github.com/topolvm/topolvm/pull/302/commits/d1651934255f87a0dc38ad2d633d8cade295668a